app/controllers/merchant/items_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Merchant::ItemsController < Merchant::BaseController 
+
+  def index
+  end
+
```

Comment:
Definitely love the way this is organized in the merchants folder.

Reply:
As do I good work here Alex